sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -399,6 +411,225 @@ public TaskManagementSteps(
                           .createdBy(tableRow.get(ColumnHeaders.CREATED_BY.toString()))
                           .build()));
         });
+
+    And(
+        "I click Export button in Task Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
+          TimeUnit.SECONDS.sleep(2);
+          webDriverHelpers.clickOnWebElementBySelector(TASK_EXPORT_BUTTON);
+        });
+
+    When(
+        "I click on the Detailed Task Export button",
+        () -> {
+          String file = "./downloads/sormas_tasks_" + LocalDate.now().format(formatter) + "_.csv";
+
+          Path file_path = Paths.get(file);
+          if (webDriverHelpers.isFileExists(file_path)) {
+            Files.delete(file_path);
+          }
+          TimeUnit.SECONDS.sleep(8); // wait for basic download if in parallel
+          webDriverHelpers.clickOnWebElementBySelector(DETAILED_EXPORT_BUTTON);
+          TimeUnit.SECONDS.sleep(8); // wait for download start
+          webDriverHelpers.waitForFileExists(file_path, 90);
+        });
+
+    When(
+        "I click on the Custom Event Export button",
+        () -> {
+          TimeUnit.SECONDS.sleep(8); // wait for basic download if in parallel
+          webDriverHelpers.clickOnWebElementBySelector(CUSTOM_EXPORT_BUTTON);
+          TimeUnit.SECONDS.sleep(2); // wait for load
+        });
+
+    When(
+        "I click on the New Export Configuration button in Custom Task Export popup",
+        () -> webDriverHelpers.clickOnWebElementBySelector(NEW_CUSTOM_EXPORT_BUTTON));
+
+    Then(
+        "I fill Configuration Name field in Custom Task Export popup",
+        () -> {
+          String configurationName = LocalDate.now().toString();
+          webDriverHelpers.fillInWebElement(
+              CUSTOM_EXPORT_CONFIGURATION_NAME_INPUT, configurationName);
+        });
+
+    When(
+        "I select specific data to export in Export Configuration for Custom Task Export",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(CUSTOM_EXPORT_TASK_CONTEXT_CHECKBOX);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_CUSTOM_EXPORT_BUTTON);
+        });
+
+    And(
+        "I open edit last created Custom Export Configuration",
+        () -> webDriverHelpers.clickOnWebElementBySelector(CUSTOM_TASK_EXPORT_EDIT_BUTTON));
+
+    And(
+        "I add specific data to export in existing Export Configuration for Custom Task Export",
```

Comment:
same, make it reusable as mentioned above and keep out as a separated method the save action